susemanager-utils/testing/automation/clean-objects.sh

```diff
@@ -0,0 +1,20 @@
+#/bin/bash -e
+echo "*************** CLEANING OBJECTS GENERATED INSIDE THE CONTAINER  ***************"
+find /manager > /tmp/objects-end.txt
+while read OBJECTEND; do
+  FOUND=FALSE
+  while read OBJECTINIT; do
+    if [ "${OBJECTINIT}" == "${OBJECTEND}" ]; then
+      FOUND=TRUE
+      break
+    fi
+  done < /tmp/objects-init.txt
```

Comment:
I think we should test if objects-init.txt exists and has content.
If not, this would remove the whole git repo.

Reply:
Right, good catch.